data/json/recipes/other/tool.json

```diff
@@ -1,4 +1,152 @@
 [
+  {
+    "result": "g_shovel",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 4,
+    "time": "3 h",
+    "autolearn": true,
+    "using": [ [ "blacksmithing_standard", 1 ], [ "steel_tiny", 1 ] ],
+    "qualities": [ { "id": "CHISEL", "level": 3 } ],
+    "tools": [ [ [ "swage", -1 ] ] ],
+    "components": [ [ [ "2x4", 1 ], [ "stick", 2 ] ] ]
+  },
+  {
+    "result": "scalpel",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
```

Comment:
How are you going to get that fine of an edge from blacksmithing? 🤔

Reply:
Agreed here.  Maybe crafting some kind of X-Acto knife level thing would be possible.

Reply:
Scalpels and cacti knives are very much an industrial process, drop the recipe.

Reply:
Removed that recipe and also X-Acto knife recipe.